processor/src/test/java/org/mapstruct/ap/test/verbose/CreateBeanMapping.java

```diff
@@ -8,7 +8,7 @@
 import org.mapstruct.Mapper;
 import org.mapstruct.factory.Mappers;
 
-@Mapper
+@Mapper( config = CreateBeanMappingConfig.class )
```

Comment:
Why do we need this test change?

Reply:
Ah.. Thats an interesting one..It printed the prism in stead of the config value :smile: